packages/mongo-menu/src/MongoMenu.tsx

```diff
@@ -133,6 +134,12 @@ const accountButtonStyle = css`
   justify-content: center;
   align-items: center;
   margin-bottom: 14px;
+
+  &:focus,
+  &:hover {
+    text-decoration: none;
```

Comment:
I think it might make more sense to apply this rule to the button component rather than here, because I imagine other LG buttons are having the same problem

Reply:
Good catch. I checked and it looks like it is in fact impacting all buttons we use that are anchors (however, that's virtually none, hence this not showing up before).